specification/specs/x_pack/ssl/get_certificates/GetCertificatesResponse.ts

```diff
@@ -17,7 +17,5 @@
  * under the License.
  */
 
-@class_serializer('GetCertificatesResponseFormatter')
-class GetCertificatesResponse extends ResponseBase {
-  certificates: ClusterCertificateInformation[]
-}
+class GetCertificatesResponse
+  implements ArrayResponseBase<ClusterCertificateInformation> {}
```

Comment:
I think this should extend from `ResponseBase` as well (cc @Mpdreamz)

Reply:
++, it should. Another possible validation @swallez :)

Reply:
What is the effect of not extending `ResponseBase`? Is this marker interface used to identify endpoint responses?

Reply:
Yeah, we want to be in a position where each api invocation can be thought of as 

`client.api(RequestBase): ResponseBase`

That way clients have a path for abstraction to

`transport.request(....) ...`